stack/schema.go

```diff
@@ -58,6 +58,14 @@ type Function struct {
 	Annotations *map[string]string `yaml:"annotations"`
 }
 
+type Configuration struct {
+	StackConfig StackConfiguration `yaml:"configuration"`
+}
+
+type StackConfiguration struct {
+	TemlpatesConfigs map[string]string `yaml:"templates"`
```

Comment:
Double plural is not very natural. what can you suggest instead?

`TemplateConfigs`?

`Temlpates` is also a typo.

Reply:
Yes, `TemplateConfig` is the way to go :+1: